beacon_detector_node:
  ros__parameters:
    enable_robot_position_reception: False
    object_topic: object
    position_topic: position_out
    debug_topic: debug
    color_topic: init/team_color
    default_color: blue
    available_colors: ["yellow", "blue"]
    beacons: [-0.094, 0.072, -0.094, 1.928, 3.094, 1., 1.275, 2.12]
    boundaries:  [0.0, 3.0, 0.0, 2.0]
    init_position: [1.77, 0.23, 0.0]
    distance_tolerance: 0.3
    angle_tolerance: 0.7853981633974483
    robot_position_topic: robot_position_input
    enable_wait_color: False
    enable_initial_position: False

lidar_simulation_node:
  ros__parameters:
    object_topic: object
    update_position_topic: update_position
    color_service: init/team_color
    default_color: blue
    beacons: [-0.094, 0.072, -0.094, 1.928, 3.094, 1., 1.275, 2.12]
    boundaries:  [0.0, 3.0, 0.0, 2.0]

nav_simulation_node:
  ros__parameters:
    moveto_action: moveto
    real_position_topic: real_position
    trigger_position_srv: trigger_position
    random_moves: False
    boundaries:  [0.0, 3.0, 0.0, 2.0]
    compute_period: 0.01
    angular_velocity: 0.4
    linear_velocity: 0.15

position_sender_node:
  ros__parameters:
    real_position_topic: real_position
    trigger_position_srv: trigger_position
    update_position_topic: update_position
    update_period: 0.03
